tests/unit/triagers/plugins/test_shipit.py

```diff
@@ -297,6 +300,33 @@ def test_supershipit_changelogs(self):
         assert sfacts[u'shipit_actors'] == []
         assert sfacts[u'shipit_actors_other'] == [u'jane']
 
+    def test_supershipit_deletion_from_sanity_ignore(self):
```

Comment:
Do we have tests to make sure that supershipit won't work when ignore files are added or have lines added/changed?

Reply:
TBD